MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -320,24 +493,56 @@ int main(int argc, char * argv[])
             {
                 if (rootDirectory.empty() || fileType == WVDFileType::NotSpecified)
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(E_INVALIDARG);
+                    errorDesc = L"Creating a file with the -create option requires both a -rootDirectory and -fileType. HRESULT " + errorCode + L".";
+
                     std::wcout << std::endl;
                     std::wcout << "Creating a file with the -create option requires both a -rootDirectory and -fileType" << std::endl;
                     std::wcout << std::endl;
+
+                    // Telemetry : Workflow Log
+                    QueryPerformanceCounter(&msixMgrLoad_EndCounter);
+                    workflowElapsedTime = msixmgr::CalcWorkflowElapsedTime(msixMgrLoad_StartCounter, msixMgrLoad_EndCounter, msixMgrLoad_Frequency);
+                    msixmgr::TraceLogWorkflow(workflowId.c_str(), cli.GetOperationTypeAsString().c_str(), false, workflowElapsedTime, errorCode.c_str(), errorDesc.c_str());
+
                     return E_INVALIDARG;
                 }
                 if (!EndsWith(unpackDestination, L".cim"))
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(E_INVALIDARG);
+                    errorDesc = L"Invalid CIM file name. File name must have .cim file extension. HRESULT " + errorCode + L".";

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.